packages/create-pantheon-decoupled-kit/src/generators/next-drupal-search-api-addon.generator.ts

```diff
@@ -21,12 +27,14 @@ export const nextDrupalSearchApiAddon: DecoupledKitGenerator<
 	addon: true,
 	data: {
 		search: true,
+		convertCssModules: true,
```

Comment:
We should be able to nix this and use the `tailwindcss`/`noTailwindcss` data prop coming from the tailwind prompt, because it will be css modules by default and tailwind (by way of conversion) otherwise.